connectors/utils.py

```diff
@@ -399,3 +401,60 @@ async def func_to_execute(*args, **kwargs):
         return func_to_execute
 
     return wrapper
+
+
+def ssl_context(certificate):
+    """Convert string to pem format and create a SSL context
+
+    Args:
+        certificate (str): certificate in string format
+
+    Returns:
+        ssl_context: SSL context with certificate
+    """
+    certificate = certificate.replace(" ", "\n")
+    certificate = " ".join(certificate.split("\n", 1))
+    certificate = " ".join(certificate.rsplit("\n", 1))
+    ctx = ssl.create_default_context()
+    ctx.load_verify_locations(cadata=certificate)
+    return ctx
+
+
+def encode(original_string):
+    """Performs encoding on the objects
+    containing special characters in their url, and
+    replaces single quote with two single quote since quote
+    is treated as an escape character
+
+    Args:
+        original_string(string): String containing special characters
+
+    Returns:
+        encoded_string(string): Parsed string without single quotes
+    """
+    encoded_string = urllib.parse.quote(original_string, safe="'")
+    return encoded_string.replace("'", "''")
+
+
+def get_expires_at(expires_in):
```

Comment:
I am concerned that this method should be in utils. Is it gonna be used somewhere else? 

a few other points: 

* The method name is misleading. What the method does is calculate a time delta. 
* You refer to `tokens` in doc strings but the method's signature knows nothing about tokens

Reply:
> I am concerned that this method should be in utils. Is it gonna be used somewhere else?

I got a comment to add it in utils.